### DISCUSSION TITLE
Macros in 6.0.0.0?

### DISCUSSION THREAD

**Discussion by tbergstedt** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#135] [Q&A] [answered]
I'm testing using the 6.0.0.0 (''alpha") version with FoundryVTT v0.8.7. Only trouble I've experienced so far is that none of the macros in the core game seem to be working. Something you authors are aware of?

I don't want to report this as an issue yet, since I don't know if the conversion has come to them yet. Besides, I'm not entirely sure I had updated the game module to the absolute latest when writing this.

**Comment by aMediocreDad** [comment, marked as answer]
Hey there! Most of the macros in the official content modules aren't working unfortunately. The table macros rely on something that saw a breaking change in 0.8. I am working on fixing all that stuff now and will send the updated modules to Free League so they can do a release. Unfortunately no ETA yet, but I really need to get them done before the end of next week (for my own sake).
